lib/edamam_api_wrapper.rb

```diff
@@ -0,0 +1,52 @@
+require 'httparty'
```

Comment:
It would be better to create a Recipe class to encapsulate the Recipe concept which lets your controller not have to be concerned with the details of the API.